### ISSUE TITLE
get syntax error : invalid syntax  in train.py at line 110

### ISSUE THREAD

**Issue by pingnuttakrid** [xuannianz/EfficientDet#127]
i got syntax error when i run train.py with my own csv data  by this command

python3 train.py --detect-quadrangle true  --snapshot imagenet  --phi 6 --gpu 0,1,2,3 --random-transform --compute-val-loss true --freeze-backbone --batch-size 32 --steps 1000 csv ./datasets/csv/train.csv ./datasets/csv/classes.csv --val-annotations ./datasets/csv/val.csv

i got syntax error on line 110 in train.py but i think it have problem in my environment not from the  code 

![image](https://user-images.githubusercontent.com/42993287/77880406-c71d6d00-7286-11ea-9d64-3a7ac06e4751.png)

**Comment by chokyungjin** [comment]
I got Same Issue!!!!!!!!!!
